solidity/test/TestGasPriceOracle.js

```diff
@@ -0,0 +1,175 @@
+const { accounts, contract, web3 } = require("@openzeppelin/test-environment")
+const { expectRevert, expectEvent } = require("@openzeppelin/test-helpers")
+const { time } = require("@openzeppelin/test-helpers")
+const { createSnapshot, restoreSnapshot } = require("./helpers/snapshot.js")
+
+const GasPriceOracle = contract.fromArtifact("GasPriceOracle");
+const GasPriceOracleConsumerStub = contract.fromArtifact("GasPriceOracleConsumerStub");
+
+const BN = web3.utils.BN
+const chai = require('chai');
+chai.use(require('bn-chai')(BN))
+const expect = chai.expect
+const assert = chai.assert
+
+describe("GasPriceOracle", () => {
+
+    const owner = accounts[0]
```

Comment:
Let's use non-default account.
```suggestion
    const owner = accounts[1]
```